models/_sector/dex/bot_trades/solana/platforms/looter_solana_bot_trades.sql

```diff
@@ -0,0 +1,145 @@
+{{
+    config(
+        alias="bot_trades",
+        schema="looter_solana",
+        partition_by=["block_month"],
+        materialized="incremental",
+        file_format="delta",
+        incremental_strategy="merge",
+        unique_key=[
+            "blockchain",
+            "tx_id",
+            "tx_index",
+            "outer_instruction_index",
+            "inner_instruction_index",
+        ],
+    )
+}}
+
+{% set project_start_date = "2023-12-26" %}
+{% set fee_receiver = "3Pu1V4duyLyVpAJue1kLAfr74nGjQ3JDzj3aJjnoEXuL" %}
+{% set wsol_token = "So11111111111111111111111111111111111111112" %}
+
+with
+    all_fee_payments as (
+        select
+            tx_id,
+            'SOL' as feetokentype,
+            balance_change / 1e9 as fee_token_amount,
+            '{{wsol_token}}' as fee_token_mint_address
+        from {{ source("solana", "account_activity") }}
```

Comment:
We can use single quotes `'` instead of `"` for `source` and `ref`:
```suggestion
        from {{ source('solana', 'account_activity') }}
```

Reply:
this is automatically inserted by sqlfmt, sorry!